WordPress/WordPressTest/Comments/Controllers/FullScreenCommentReplyViewModelTests.swift

```diff
@@ -0,0 +1,25 @@
+
+import XCTest
+@testable import WordPress
+
+class FullScreenCommentReplyViewModelTests: CoreDataTestCase {
+    private var suggestionsServiceMock: SuggestionServiceMock!
+    private var sut: FullScreenCommentReplyViewModel!
+    private let number = NSNumber(value: 1)
+
+    override func setUp() {
+        super.setUp()
+        suggestionsServiceMock = SuggestionServiceMock(context: mainContext)
+        sut = FullScreenCommentReplyViewModel(suggestionsService: suggestionsServiceMock, context: mainContext)
+    }
+
+    func testSuggestionsTableViewIsNotNil() {
+        let tableView = sut.suggestionsTableView(with: number, useTransparentHeader: false, prominentSuggestionsIds: [number], delegate: SuggestionsTableViewDelegateMock())
+
+        XCTAssertNotNil(tableView)
+    }
+
+    func testShouldShowSuggestionsIsFalse() {
```

Comment:
This test is a bit vague. Can you either add a more verbose setup in the test or add some documentation?

Reply:
Sure. I added a more verbose setup. Is it ok like that?